src/components/AppHeader.tsx

```diff
@@ -34,21 +34,21 @@ interface AppHeaderCallbackProps {
 };
 
 const mapStateToProps = (state: XCSpyState): AppHeaderProps => {
+    const initialized = state.components.initialized;
+    const components = ((): string[] => {
```

Comment:
Why do you need to create an anonymous function ? 
Replace by const components = Initialized ?  Object.keys(state.components.componentProperties) : [];